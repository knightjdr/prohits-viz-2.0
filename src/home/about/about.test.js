import React from 'react';
import { shallow } from 'enzyme';

import About from './about';

describe('About', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(
      <About />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
