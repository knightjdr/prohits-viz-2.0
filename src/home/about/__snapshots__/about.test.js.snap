// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should match snapshot 1`] = `
<footer
  className="about"
>
  <div
    className="about__funding"
  >
    <p>
      Funding for this project was graciously provided by:
    </p>
    <div
      className="about__funding-wrapper"
    >
      <a
        className="about__funding-link"
        href="http://www.cihr-irsc.gc.ca/"
        key="CIHR"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="CIHR"
          height="50px"
          src="CIHR.svg"
        />
      </a>
      <a
        className="about__funding-link"
        href="https://www.genomecanada.ca/"
        key="Genome Canada"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Genome Canada"
          height="50px"
          src="GenomeCanada.svg"
        />
      </a>
      <a
        className="about__funding-link"
        href="http://www.ontariogenomics.ca/"
        key="Ontario Genomics"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="Ontario Genomics"
          height="50px"
          src="OntarioGenomics.svg"
        />
      </a>
      <a
        className="about__funding-link"
        href="http://www.lunenfeld.ca/"
        key="LTRI"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="LTRI"
          height="50px"
          src="LTRI.svg"
        />
      </a>
    </div>
  </div>
  <div
    className="about__footer"
  >
    Â© 2018,
    <a
      className="about__link"
      href="http://gingraslab.lunenfeld.ca/"
      rel="noopener noreferrer"
      target="_blank"
    >
      Gingras lab
    </a>
  </div>
</footer>
`;
